thoth/storages/graph/dgraph.py

```diff
@@ -310,7 +310,25 @@ def runtime_environment_analyses_listing(
         self, runtime_environment_name: str, start_offset: int = 0, count: int = 100
     ) -> list:
         """Get listing of analyses available for the given environment."""
-        return []
+        query = """
+        {
+            f(func: has(%s), first: %d, offset: %d) @filter(eq(environment_name,"%s")){
+			    count:count(environment_name)
+                analyzed_by{
+                    analysis_datetime
+                  	analysis_document_id
+                  	package_extract_name
+                  	package_extract_version
+                }
+            }   
+        }
```

Comment:
There are mixed tabs and spaces. Please use just spaces otherwise coala will complain.